data/monitoring/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    prometheus.cnao.io: ""
+  name: cluster-network-addons-operator-prometheus-metrics
+  namespace: {{ .Namespace }}
+spec:
+  ports:
+    - name: metrics
+      port: 443
```

Comment:
If we are using the default 8080 would be good also since we are not using TLS 443 does not make sense here.